main.js

```diff
@@ -0,0 +1,211 @@
+console.log('hi');
+
+var mainTimer = document.querySelector('.displayed-time');
+var currentWork = document.querySelector('.current-work');
+var input = document.querySelector('input[name="input-value"]');
+var workingStuff = document.querySelector('.working-stuff');
+var start = document.querySelector('button[name="start"]');
+var form = document.querySelector('form');
+var inputArea = document.querySelector('.input-area');
+var bodyWrapper = document.querySelector('.body-wrapper');
+var contentWrapper = document.querySelector('.content-wrapper');
+var hiddenBreak = document.querySelector('.quick-break');
+var h2 = document.querySelector('.question');
+var myAudio = new Audio('expired-timer.mp3');
+
+var inputValue = null;
+var heldInput = null;
+
+var minutesNumber = 24; //24
+var secondsNumber = 59;  //59
+
+var intervalMinId;
+var intervalSecId;
+var intervalDisId;
+
+var breakMinutes = 4;
+var breakSeconds = 59;
+
+var breakMinId;
+var breakSecId;
+var breakDisId;
+
+var audioTimeoutId;
+
+input.focus();
+
+//start timer
+var handleClick = function(evt) {
+  if (input.value === '' /* || +input.value !== NaN*/){
+    input.classList.add('error');
+    input.placeholder = "Enter a task, scrub...";
+  } else {
+      setInt();
+      displayTimer();
+      start.removeEventListener('click', handleClick);
+      input.classList.remove('error');
+      getNClear();
+      showContentTimer();
+      hideInputArea();
+    }
+}
+
+start.addEventListener('click', handleClick);
+
+var setInt = function(){
+  intervalMinId = setInterval(minCount, 60*1000 / 120);  // shortened for testing
+  intervalSecId = setInterval(secCount, 1000 / 120);
+  intervalDisId = setInterval(displayTimer, 1000 / 120);
+}
+
+var getNClear = function(){
+  heldInput = input.value;
+  toTitleCase(input.value);
+  workingStuff.innerHTML = 'Currrently working on: <span class="current-work">' + input.value+ '</span>';
+  // currentWork.textContent = input.value;
+  input.value = '';
+}
+
+var displayTimer = function() {
+  // minutesNumber = minutesNumber < 10 ? "0" + minutesNumber : minutesNumber;
```

Comment:
What's happening is you're saying If the number is less than 10 assign the variable to "0" + the number, which turns the whole thing into a string, so the next time you're checking to see if a string is less than 10.
The thinking makes sense, but you wouldn't want to reassign `minutesNumber` try using another variable like `displayMin = minutesNumber < 10 ? "0" + minutesNumber : minutesNumber`